src/main/java/com/ttbkk/api/domain/hashtag/Hashtag.java

```diff
@@ -0,0 +1,30 @@
+package com.ttbkk.api.domain.hashtag;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+
+//Entity 클래스
+@Getter
+@NoArgsConstructor
+@Entity
+@Table(name = "hashtag")
+public class Hashtag {
+
+    @Id
+    @Column(columnDefinition = "VARCHAR(150)")
+    private String name;
+
+    @NotNull
+    @Column(name = "created_at", columnDefinition = "DATETIME(6)")
+    private LocalDateTime createdAt;
```

Comment:
BaseTimeEntity에는 없던 NoNull이 여기에는 붙어있네요?
그리고 CreatedDate 어노테이션이 필요해보입니다

```suggestion
    @NotNull
    @CreatedDate
    @Column(name = "created_at", columnDefinition = "DATETIME(6)")
    private LocalDateTime createdAt;
```

Reply:
@createdDate 어노테이션으로 사용하여 시간을 자동생성하기 위해선 basetimeentity 같은 entity리스너 클래스를 상속 받아 audit 로 자동 시간을 할당 받아야 생성 되기 때문에 basetimeentity 를 상속받으면 생성 시간 뿐만 아니라, 수정시간 까지 관리 하게 되어 따로 생성자에서 처리 시켰습니다. 말씀 하신 @createddate 어노테이션으로 추가해서 테스트를 돌려 봤는데 시간이 null 이 나와서 어떤 방법으로 하는게 좋을까요?

Reply:
그렇다면 createdAt만 있는 BaseEntity를 따로 두는게 맞을것같네요
추후에는 updatedAt 컬럼을 추가해서 하나의 BaseEntity로 관리하는게 좋을것같긴 합니다

Reply:
https://github.com/ttbkk-univ/ttbkk-spring/pull/7/commits/6ae91281d929fe9ba75119f7bb9230a74366efc0